src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -504,7 +504,8 @@ public BoundAsOperator As(BoundExpression operand, TypeSymbol type)
         public BoundIsOperator Is(BoundExpression operand, TypeSymbol type)
         {
             HashSet<DiagnosticInfo> discarded = null;
-            Conversion c = Compilation.Conversions.ClassifyConversionFromExpression(operand, type, ref discarded);
+            const bool preferBuiltinConversion = true;
+            Conversion c = Compilation.Conversions.ClassifyConversionFromExpression(operand, type, ref discarded, preferBuiltinConversion);
```

Comment:
>preferBuiltinConversion [](start = 114, length = 23)

It doesn't look like passing this value guarantees that user defined conversions are ignored completely.

Reply:
That isn't the intended behavior. It just needs to prioritize builtin conversions over user-defined conversions.

Reply:
>  It just needs to prioritize builtin conversions over user-defined conversions.

What would be a valid scenario when a user defined conversion should be accepted here?

Reply:
I don't know what you mean by "accepted". The conversion is required in the node but isn't used subsequent to lowering. The `is` operator in the language does not actually require that a conversion exist. The conversion is stored to assist with semantics (specifically warnings) and lowering when the `is` operator appears in source. Those don't apply to the node produced here.

If it would make it more clear, we can use `NoConversion` here.

Reply:
Or, if you prefer, we can compute a built-in conversion from type.